ext/oauth-client/Civi/Api4/Action/OAuthSysToken/Refresh.php

```diff
@@ -38,10 +38,15 @@ class Refresh extends BasicBatchAction {
 
   private $syncFields = ['access_token', 'refresh_token', 'expires', 'token_type'];
   private $writeFields = ['access_token', 'refresh_token', 'expires', 'token_type', 'raw'];
+  private $selectFields = ['id','client_id','access_token', 'refresh_token', 'expires', 'token_type', 'raw'];
```

Comment:
Hi @ufundo the test fail is complaining about the formatting. It likes a space after commas.

```
Refresh.php:41,           NoSpace, Priority: High
--
Expected one space after the comma, 0 found
```